module/VuFind/src/VuFind/Auth/Database.php

```diff
@@ -46,8 +46,8 @@ class Database extends AbstractBase
-    public static $sexes = ["Mr", "Ms"];
-    public static $titles = ["B.A.", "M.A.", "M.Div.", "Dipl. Theol.", "Dr.", "Ph.D.", "Th.D.", "Prof.", "Lic. theol.", "Lic. iur. can.", "Student", "Other",];
+    public static $sexes = ["Mr", "Ms", ""];
```

Comment:
"sexes" is not what one would call these.  I suggest "appellations" or "courtesy_titles".